examples/spice.mzLib.txt

```diff
@@ -0,0 +1,793 @@
+<mzSpecLib 1.0>
+MS:1003188|library name=
+MS:1003190|library version=1.0
+MS:1001017|release date=7/7/2023 7:06:42 PM
+<Spectrum=1>
+MS:1003061|library spectrum name=JWH-250-5OH
+MS:1003062|library spectrum index=795
+MS:1000894|retention time=3.53813
+MS:1000864|chemical formula=
+MS:1002894|InChiKey=
+MS:1009100|CASNo=
+MS:1000868|Smiles formula=
+MS:1009101|compound class=
+MS:1003059|number of peaks=51
+<Analyte=1>
+MS:1000744|selected ion m/z=352.19000

```

Comment:
There is a preference for `MS:1003208|experimental precursor monoisotopic m/z` over `selected ion m/z`, and I think `selected ion m/z` is still allowed but only in the `Spectrum` section. @henryhlam would be able to say better.

Reply:
Changed to MS:1003208|experimental precursor monoisotopic m/z.